client/bootstrap/createApolloClient.ts

```diff
@@ -7,7 +8,9 @@ import {
 } from '../shared/graphql/__generated__/Page.graphql';
 import { isSSR } from '../shared/helpers/isSSR';
 
-export function createApolloClient() {
+export function createApolloClient(): {
+  persist: (callback: (client: ApolloClient<unknown>) => void) => void;
```

Comment:
[IMO]
`persist` is implementation detail and caller should not know whether the data will be persisted or not.
`initialize` would be a better option.